.travis.yml

```diff
@@ -1,6 +1,7 @@
 conditions: v1
 stages:
   - name: lint
+  - name: mypy
```

Comment:
Since stages run sequentially on travis it's probably better to add this to the `lint` stage so we don't unnecessarily increase test runtime (even though travis is on the way out anyway).

Reply:
Good idea, will do in this PR.

Reply:
Done!